docs/orchestration/concepts/kv_store.md

```diff
@@ -0,0 +1,123 @@
+# KV Store
+
+Key Value Store is your managed metadata database within Prefect Cloud.
+
+**Keys** are strings. **Values** are json blobs.
+
+The number of key value pairs allowed is limited by license, starting with 50 pairs on the Free tier. Values are limited to 1 MB in size.
+
+Key value pairs can be configured via the Prefect CLI, Python client, API, and UI.
+
+## UI
+
+You can view, update, and delete key value pairs on the [KV Store page](https://cloud.prefect.io/team/kv) of the UI.  
+
+## Setting Key Value Pairs
+
+Setting a key value pair will overwrite the existing value if the key exists.
+
+:::: tabs
+::: tab Core
```

Comment:
I did this in my docs and it makes more sense than our `Core` lingo I think
```suggestion
::: tab Prefect library
```

Reply:
(later too)

Reply:
I also did `GraphQL API` instead of `GraphQL` -- thoughts?

Reply:
works for me, updated